### PR TITLE
fix for 0.5

### PR DESCRIPTION
**Pull request by sbromberger**

### CODE REVIEW

File: src/IPNets.jl [hunk 1]
```diff
@@ -1,11 +1,9 @@
-VERSION >= v"0.4.0-dev+6521" && __precompile__(true)
```
Comment (tkelman) [inline src/IPNets.jl]: probably shouldn't disable precompilation

Comment (sbromberger) [inline src/IPNets.jl]: I thought precompilation was default in 0.5?

Comment (tkelman) [inline src/IPNets.jl]: It isn't.

Comment (sbromberger) [inline src/IPNets.jl]: Ah, crap. Ok. I'll close this out and retag.

**Pull request merged**